Add Gitter badge to the README (#227)
aesara-devs/aeppl: README.md

## Get help

Report bugs by opening an [issue][issues]. If you have a question regarding the usage of AePPL, start a [discussion][discussions]. For real-time feedback or more general chat about AePPL use our [Discord server][discord], or [Gitter room][gitter].

## Contribute

> AePPL welcomes contributions. A good place to start contributing is by looking at the [issues][issues].

Reviewer: ```suggestion
AePPL welcomes contributions. To start contributing, take a look at the open [issues][issues].
```
